### ISSUE TITLE
The usage is not correct

### ISSUE THREAD

**Issue by 13724345594** [danielgindi/Charts#1521]
" Drag the Charts.xcodeproj to your project ", maybe no one can successfully compile the project.  Please correct the use of instructions, please try to detail a little. Just think the user as a new iOS developer, thanks!

**Comment by Huang-Libo** [comment]
@13724345594 
The usage has no problem at all.

**Comment by petester42** [comment]
A new iOS developer should probably learn how the tool (Xcode) works and gain experience with it before starting to mess with third-party frameworks. If the instructions are hard to follow it is not our fault that is how Xcode works. To me if you are looking for an easy way to add this to your project I would put Cocoapods as the easiest way, then Carthage and then manually adding Charts to your project.

I usually use cocoapods for my projects since it makes it very simple.

**Comment by 13724345594** [comment]
@Huang-Libo, @petester42 
Thanks a lot , the problem has been solved, but not with the author's method, just feel the author provides the method is not very detailed

**Comment by petester42** [comment]
To be clear it is not the author's method. It is the method you need to use when using dynamic frameworks. If you have a more detailed instructions feel free to send us a pull request!
